membership-attribute-service/app/models/SupporterRatePlanItem.scala

```diff
@@ -0,0 +1,10 @@
+package models
+
+import org.joda.time.LocalDate
+
+case class SupporterRatePlanItem(
+    identityId: String, //Unique identifier for user
+    ratePlanId: String, //Unique identifier for this product purchase for this user
+    productRatePlanId: String, //Unique identifier for the product in this rate plan
+    termEndDate: LocalDate //Date that this subscription term ends
+)
```

Comment:
this is the dynamo format for the item I guess? it's not clear from the name

Reply:
Have updated the name to make it clearer